Example/Stagehand/CGAffineTransformDebuggingViewController.swift

```diff
@@ -0,0 +1,345 @@
+//
```

Comment:
This VC was super helpful in debugging issues with transforms. I filed #32 to reorganize the demo app, since this feels a bit out of place with some of the other screens that show off features of the framework.